src/aatoolbox/datasources/fewsnet/fewsnet.py

```diff
@@ -27,243 +32,229 @@
-        region_data = _download_zip(
-            url=url_region_date,
-            zip_filename=zip_filename_region,
-            output_dir=output_dir_region,
+        # FN data can be regional, and thus we save the raw data
+        # in "glb" instead of in the iso3 folder, even though part of it is at
+        # country level
+        self._raw_base_dir = self._get_base_dir(
+            is_public=True,
+            is_raw=True,
+            is_global=True,
         )
-    else:
-        region_data = output_dir_region
-    return region_data
 
+        self._iso2 = Country.get_iso2_from_iso3(self._country_config.iso3)
+        if self._iso2 is None:
+            logger.error("No ISO2 found for the given ISO3. Check your ISO3.")
 
-def _download_zip(
-    url: str,
-    zip_filename: str,
-    output_dir: Path,
-) -> Optional[Path]:
-    """
-    Download and unzip the file at the url.
+    def download(
+        self,
+        date_pub: Union[date, str],
+        clobber: bool = False,
+    ) -> Path:
+        """
+        Retrieve the raw fewsnet data.
 
-    Parameters
-    ----------
-    url : str
-        url that contains the zip file to be downloaded
-    zip_filename : str
-        name of the zipfile
-    output_dir : Path
-        path of dir to which the zip content should be written
+        Depending on the region and date, this data is published per region or
+        per country. This function retrieves the country data
+        if it exists, and else the regional data for `date_pub`.
 
-    Returns
-    -------
-    None if no valid file, else output_dir
+        Parameters
+        ----------
+        date_pub : date or str
+            date for which the data should be downloaded
+            only the year and month part are used
+            this commonly refers to the month of the Current Situation period
+            if str, it should be in ISO 8601 format, e.g. '2021-10-01'
+        clobber : bool, default = False
+            If True, overwrites existing raw files
 
-    """
-    # create tempdir to write zipfile to
-    with TemporaryDirectory() as temp_dir:
-        zip_path = Path(temp_dir) / zip_filename
-        download_url(url=url, save_path=zip_path)
-        logger.info(f"Downloaded {url} to {zip_path}")
-
-        try:
-            unzip(zip_file_path=zip_path, save_dir=output_dir)
-            logger.debug(f"Unzipped to {output_dir}")
-
-        except zipfile.BadZipFile:
-            # indicates that the url returned something that wasn't a
-            # zip, happens often and indicates data for the given
-            # country - date is not available
-            logger.debug(
-                f"No zip data returned from url {url} "
-                f"check that the area and date exist."
+        Examples
+        --------
+        >>> download(date_pub="2020-10-01")
+        """
+        if (
+            self._country_config.fewsnet.region_name  # type: ignore
```

Comment:
I think I had this problem also with GloFAS. If I recall correctly, the way to fix it (which is actually the correct thing to do) is to check and warn / exit if `country_config.fewsnet` is `None` i.e. not implemented